### PR TITLE
Wrong reference

### PR DESCRIPTION
**Pull request by danielthorvil**
I think line 249 is meaning to reference to the "when" functionality, not the "on_failure" functionality.

Fixes #

## Target Release

1.8.x

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS

 

-

### CODE REVIEW

**Comment by crw** [comment]
Thanks for this submission!

**Comment by crw** [comment]
Given the way the rest of this document is written, I believe the original text is actually what is intended (even though the next section is on `when`, there is in fact a section farther down describing `on_failure`). If you think this is incorrect, please let me know and I will also run this past another person on the team just to check my thinking. Thanks!

File: website/docs/language/resources/provisioners/syntax.mdx
Comment (trujillo-adam) [inline website/docs/language/resources/provisioners/syntax.mdx:249]:
```suggestion
You can change this behavior by setting the `on_failure` attribute to `continue`. Refer to [Failure Behavior](#failure-behavior) for additional information.
```
Per the style guide, we should use specific references instead of directional terms, such as "below" and "above".

File: website/docs/language/resources/provisioners/syntax.mdx
Comment (trujillo-adam) [inline website/docs/language/resources/provisioners/syntax.mdx:250]:
```suggestion
```

**Review by trujillo-adam — COMMENTED** [review]
Thanks for calling attention to this. It is very easy to see how the original text is ambiguous. We need to do a better job of following our own style guide. I think these suggestions will clear things up.

**Review by crw — APPROVED** [review]
Approving after merging trujillo-adam's changes

**Comment by crw** [comment]
Hi @danielthorvil, I went ahead and committed those changes and approved this PR based on the new changes. I am going to merge this PR, if you feel there is anything else you want to change please feel free to open a new PR. Thanks very much!

**Pull request merged**
